src/altinn-app-frontend/src/features/expressions/shared.test.ts

```diff
@@ -48,7 +48,8 @@ describe('Expressions shared function tests', () => {
             return evalExpr(expr as Expression, component, dataSources);
           }).toThrow(expectsFailure);
         } else {
-          expect(evalExpr(expression, component, dataSources)).toEqual(expects);
+          const expr = asExpression(expression) as Expression;
+          expect(evalExpr(expr, component, dataSources)).toEqual(expects);
```

Comment:
I'm doing this so that the test fails with a thrown exception in case any validation errors occur. They shouldn't, but it's nice to get that error instead of a silently working test. Should have been there all the time.